rfd/0018/README.md

```diff
@@ -76,7 +77,11 @@ What we propose here is to make a reserved namespace for labels that have
 special meanings for sdc-docker. These would look like:
 
 ```
-com.joyent.<key>=<value>
+com.joyent.<key>=<value>     # Joyent/Triton-specific use cases
+```
+or
+```
+triton.<key>=<value>         # Triton-specific use cases
```

Comment:
Is this suggesting both are supported, or is this posing a question for us to decide on?